test/JDBC/upgrade/13_9/schedule

```diff
@@ -97,7 +97,6 @@ BABEL-889
 BABEL-APPLOCK
 babel_char
 BABEL-CHECK-CONSTRAINT-before-14_6
-BABEL-CROSS-DB
```

Comment:
Yeah cause cross database references was not supported in 13.x so prepare file has become incompatible for those versions so I removed the test from 13.x schedules files since we anyway don't support a lot of things their.